cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -887,8 +891,9 @@ def on_gpu_available(kube_control):
     We need to run in privileged mode.
 
     """
+    kube_version = get_version('kube-apiserver')
     config = hookenv.config()
-    if config['allow-privileged'].lower() == "false":
+    if config['allow-privileged'] == "false" and kube_version < (1, 9):
```

Comment:
Where'd the `.lower()` go? It was originally introduced in https://github.com/kubernetes/kubernetes/pull/59116 as a fix for https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/374

Reply:
Probably missed it during a rebase :(